tests/k8s/deploy-services.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -euo pipefail
+
+# Script to deploy vault-manager test services in a local Kubernetes cluster
+# Usage: ./deploy-services.sh [namespace]
+#
+# Requirements:
+# - kubectl or oc CLI
```

Comment:
is oc supported? I only see kubectl commands below. It could be an env var

Reply:
Yes, just force of habit using kubectl.  I don't think an env var will be necessary - updating this to just use oc.